### PR TITLE
Update endpoint-dlp-using.md

### PR DESCRIPTION
**Pull request by msbemba**
Added description for Unallowed Bluetooth apps and Always audit file activity for devices.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5188

File: microsoft-365/compliance/endpoint-dlp-using.md [hunk 1]
```diff
@@ -92,6 +95,11 @@ You can control how users interact with the business justification option in DLP
 - Users can only select a built-in justification.
 - Users can only enter their own justification.
 
+### Always audit file activity for devices
+
+By default, when devices are onboarded, activity for Office, PDF, and CSV file is automatically audited and available for review in activity explorer. Turn this feature off if you want this activity to be audited only when onboarded devices are included in an active policy.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-using.md]:
```suggestion
By default, when devices are onboarded, activity for Office, PDF, and CSV files is automatically audited and available for review in activity explorer. Turn this feature off if you want this activity to be audited only when onboarded devices are included in an active policy.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - minor edit.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**